packages/nouns-contracts/test/governance/NounsDAO/V3/voteRefund.test.ts

```diff
@@ -0,0 +1,384 @@
+const LONG_REASON = realLongReason + realLongReason;
+const REFUND_ERROR_MARGIN = ethers.utils.parseEther('0.001');
```

Comment:
FYI @solimander @davidbrai I once again made the test less sensitive; it used to be `0.00015` and the latest code fails to reach this level of accuracy, I'm guessing because of the changes we did around voting in V3, and I don't think it's important to deep dive into this. wdyt?